fuzz/README.md

```diff
@@ -0,0 +1,94 @@
+# ruff-fuzz
+
+Fuzzers and associated utilities for automatic testing of Ruff.
+
+## Usage
+
+To use the fuzzers provided in this directory, start by invoking:
+
+```bash
+./fuzz/init-fuzzers.sh
+```
+
+This will install [`cargo-fuzz`](https://github.com/rust-fuzz/cargo-fuzz) and optionally download a
+[dataset](https://zenodo.org/record/3628784) which improves the efficacy of the testing.
+The dataset may take several hours to download and clean, so if you're just looking to try out the
+fuzzers, skip the dataset download, though be warned that some features simply cannot be tested
```

Comment:
How can I skip the dataset download or does the script ask me whether to download the dataset?

Reply:
The script asks :slightly_smiling_face: